sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -1608,6 +1608,8 @@ def is_in_terminal_state(self):
     return PipelineState.is_terminal(self._get_job_state())
 
   def wait_until_finish(self, duration=None):
+    consoleUrl = """Console URL: 'https://console.cloud.google.com/
+      dataflow/jobs/<RegionId>/{}?project=<ProjectId>'""".format(self.job_id)
```

Comment:
Big thank you for helping with this issue. Seeing a formatting error:

```
AssertionError: Job did not reach to a terminal state after waiting indefinitely.Console URL: 'https://console.cloud.google.com/ |  
-- | --
  | dataflow/jobs/<RegionId>/<bound method DataflowPipelineResult.job_id of <DataflowPipelineResult <Job |  
  | clientRequestId: '20220504201710949356-2017'
```
self.job_id() ?

Reply:
PTAL at other places as well. Thank you!